src/core/qobj.jl

```diff
@@ -1766,3 +1777,66 @@ function genlaguerre(x, alpha, n)
     end
     return result
 end
+
+function get_canonical_global_phase(H::AbstractMatrix{Complex{T}})::T where {T<:Real}
```

Comment:
one more place we should use this function is inside `as_universal_gate()`

Reply:
Updated